docs/services/overview-pages.js

```diff
@@ -51,39 +55,55 @@ function createPages(obj, slug) {
   });
 }
 
-const renderOverviewPages = (allPages, callback) => {
-  allPages.forEach(page => {
-    const { pages } = page;
-    if (!pages) return undefined;
-    callback(page);
-    return renderOverviewPages(pages, callback);
-  });
+const forEachNestedPage = (pages, callback) => {
+  pages
+    .filter(page => page.pages)
+    .forEach(page => {
+      callback({
+        ...page,
+        // keep pages nested only one level deep
+        pages: page.pages.map(p => _.omit(p, ["pages"])),
+      });
+      forEachNestedPage(page.pages, callback);
+    });
 };
 
-async function createOverviewPages(callback) {
+async function getOverviewPages() {
```

Comment:
I renamed this function so that it better describes that it now returns an array of overview pages.